src/adapter/etl-adapter-xml/composer.json

```diff
@@ -14,7 +14,9 @@
         "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
         "ext-dom": "*",
         "ext-xmlreader": "*",
-        "flow-php/etl": "^0.5.0 || 1.x-dev"
+        "ext-xmlwriter": "*",
+        "flow-php/etl": "^0.5.0 || 1.x-dev",
+        "veewee/xml": "^2.13.0"
```

Comment:
we should aim to support lowest version of the lib, so if none of our features are using things added after 2.0.0 we should rather set this dependency to be `^2.0`, unless we are using something added/fixed in `2.13.0.` then `^2.13.0`

Reply:
Removed usage of the library in favor of direct playing with `DOMDocument` & `XMLWriter`.